docs/tutorials/contributing/strategy/writing_the_new_strategy.rst

```diff
@@ -115,30 +117,19 @@ opponent.history[-1]`)::
         # Repeat the opponent's last move
         return opponent.history[-1]
 
-The variables :code:`C` and :code:`D` represent the cooperate and defect actions respectively.
+The variables :code:`C` and :code:`D` represent the cooperate and defect actions
+respectively.
 
-You can also modify the name of the strategy with the :code:`__repr__` method,
-which is invoked when :code:`str` is applied to a player instance. For example,
-the :code:`Random` strategy takes a parameter :code:`p` for how often it
-cooperates, and the :code:`__repr__` method adds the value of this parameter to
-the name::
+Some strategies make specific use of the variables of a match to create their
```

Comment:
Maybe a bit more detail here. After:

+Some strategies make specific use of the variables of a match to create their
+own attributes.

Then:

In principle these attributes could change throughout a match or tournament if the match properties (like the game matrix) change, so we require that this logic live in the 
the :code:`receive_match_attributes` method for correct dynamic updating. Here is how this is done for
+:class:`Stalker <axelrod.strategies.stalker.Stalker>`:

Reply:
Done! :+1: